play/PlayerRules/CharacterCreation/05_ArmourList.txt

```diff
@@ -165,386 +203,284 @@ Cost For Arm or Leg Armor:	2900
 APL (Armor Piercing Level):	10
 APs (Armor Points):		130
 
+	
+    =====================================
+		 ARMOR RIGS
+    =====================================
+Armor rigs cannot be worn in combination with armor
+other rigs, plate carriers, soft armour, or tactical vests.
+	
+	
+	
+*michael halp me plsz* I took the armor descriptions from
+1.0 armor and put it below:

```

Comment:
Why keep it if you have more flavor text in the new stuff?

Reply:
I see that most of the flava has been removed. I kinda like to keep flavor in a per-item basis anyway. nuke it.

Reply:
I don't have much flavor text in the new stuff. But, for now, I think it's good to have some flavor text for each armor group (recon, light, medium, w/e) so that players understand some of the lore, that we aren't running around in halo armor